### ISSUE TITLE
Feature request: Support for note blocks

### ISSUE THREAD

**Issue by penenkel** [madskristensen/MarkdownEditor#187]
As the title says, it would be nice to have support for note blocks (e.g. NOTE, WARNING, ...).

For a more detailed description see the corresponding feature from Microsoft Docs: [link](https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#note-blocks)

**Comment by madskristensen** [comment]
Where does that syntax render those blocks as shown in the Microsoft Docs? Does GitHub support it or is it a Microsoft Docs concept?

**Comment by penenkel** [comment]
Yeah, upon further investigation, it seems like this is a custom MS Docs concept 🙁. GitHub has no equivalent that I could find. GitLab has something [similar](https://about.gitlab.com/handbook/product/technical-writing/markdown-guide/#colorful-sections) but that also looks like a custom feature. Nevertheless, it would be nice to have 😄.
